homeassistant/components/keba/__init__.py

```diff
@@ -0,0 +1,212 @@
+"""Support for KEBA charging stations."""
+import asyncio
+import logging
+
+from keba_kecontact.connection import KebaKeContact
+import voluptuous as vol
+
+from homeassistant.const import CONF_HOST
+from homeassistant.helpers import discovery
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'keba'
+SUPPORTED_COMPONENTS = ['binary_sensor', 'sensor', "lock"]
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(CONF_HOST): cv.string,
+        vol.Optional('rfid', default='00845500'): cv.string,
```

Comment:
Please break out constants for the strings that are used multiple times.

Reply:
done